app/src/main/res/layout/recent_bookmarks_header.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-
 <androidx.constraintlayout.widget.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginTop="@dimen/recent_bookmark_header_margin_top">
+    android:layout_marginHorizontal="@dimen/home_item_horizontal_margin"
```

Comment:
nit: Not sure what we want going, but should we be mixing dimens and dp?  Maybe we should just use dp directly? or defined all in dimens?

Reply:
We spoke about this over Slack, but overall this is fine since it's utilizing an existing dimens variable that is shared across multiple views vs using the dp directly.